tools/src/main/java/com/nedap/archie/rmobjectvalidator/RMObjectValidationMessage.java

```diff
@@ -95,7 +105,9 @@ public void setType(RMObjectValidationMessageType type) {
     /**
      * Get the human readable path in the archetype that this validation refers to - used to retrieve the constraint that was used to generate this error
      * @return the human readablepath of the constraint that was used to generate this error in the archetype
+     * @deprecated This functionality will be removed.
```

Comment:
This could still be somewhat useful to display errors in a way humans can understand. Although the official syntax for this could instead be used here: `/items[id3 and name='blood pressure']`